pennylane/transforms/draw.py

```diff
@@ -274,38 +274,43 @@ def circuit2(x, y):
 
         **Formatting:**
 
-        You can globally control the style with ``plt.rcParams`` and styles, see the
-        `matplotlib docs <https://matplotlib.org/stable/tutorials/introductory/customizing.html>`_ .
-        If we customize ``plt.rcParams``, we get a
-        different style:
+        PennyLane has inbuilt styles. All available styles can be determined by
+        evaluating ``qml.drawer.available_styles()``. Any available string
+        can then be passed to ``qml.drawer.use_style``.
 
         .. code-block:: python
 
-            plt.rcParams['patch.facecolor'] = 'white'
-            plt.rcParams['patch.edgecolor'] = 'black'
-            plt.rcParams['patch.linewidth'] = 2
-            plt.rcParams['patch.force_edgecolor'] = True
-            plt.rcParams['lines.color'] = 'black'
-
+            qml.drawer.use_style('black_white')
             fig, ax = qml.draw_mpl(circuit)(1.2345,1.2345)
             fig.show()
 
-        .. figure:: ../../_static/draw_mpl/rcparams.png
+
+        .. figure:: ../../_static/draw_mpl/black_white_style.png
                 :align: center
                 :width: 60%
                 :target: javascript:void(0);
 
-        Instead of manually customizing everything, you can choose one of
-        the provided styles. You can see available styles with ``plt.style.available``.
-        We can set the ``'Solarize_Light2'`` style and instead get:
+        You can globally control the style with ``plt.rcParams`` and styles
+        If we customize ``plt.rcParams``, we get a
+        different style:

```

Comment:
The wording sounded a little weird. Is *styles* a separate parameter used with ``plt.rcParams``? If not I think this suggestion sounds better: 
 
```suggestion
        You can globally control the style with ``plt.rcParams``.
        If we customize ``plt.rcParams``, we get a different style:

```